style.css

```diff
@@ -442,3 +462,393 @@ ul > li > i {
+    align-items: flex-start;
+    padding: 24px 22px;
+  }
+
+  .project-title {
+    width: 496px;
+    height: 46px;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    margin: 20px;
+  }
+
+  .language-list {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 297px;
+    height: 40px;
+  }
+
+  .credentials-list {
+    height: 40px;
+    align-items: center;
+  }
+
+  .language-list li {
+    align-items: center;
+    background-color: #ebebff;
+    border-radius: 8px;
+    font-weight: 500;
+  }
+
+  .description {
+    width: 496px;
+    height: 50px;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .about-myself {
+    width: 100%;
+    height: 924px;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .myself-left-block {
+    width: 447px;
+    height: 114px;
+    margin-top: 142px;
+    margin-left: 142px;
+  }
+
+  .about-myself-title {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .my-profile {
+    width: 447px;
+    height: 102px;
+    font-size: 16px;
+    line-height: 24px;
+    margin: 12px 0;
+    text-align: justify;
+  }
+
+  .contact-me-icons {
+    width: 200px;
+    height: 40px;
+    margin-left: -240px;
+  }
+
+  .skill-list {
+    width: 684px;
+    height: 360px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 142px;
+    margin-left: 25px;
+    margin-right: 142px;
+  }
+
+  .frame {
+    width: 684px;
+    border-bottom: 1px solid #dfe1e6;
+    padding: 24px 0;
+  }
+
+  .frame-summary {
+    list-style-type: none;
+    padding-left: 12px;
+    background-image: url(images/chevron-right.svg);
+    background-repeat: no-repeat;
+    background-position: 600px;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 20px;
+    line-height: 24px;
+  }
+
+  .contact-me {
+    width: 100%;
+    height: 846px;
+    background-color: #6070ff;
+  }
+
+  .contact-me h2 {
+    width: 801px;
+    height: 56px;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    align-items: center;
+    margin: 10px 320px 0 320px;
+  }
+
+  .contact-me p {
+    width: 569px;
+    height: 102px;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+    margin: 12px 435px 12px 436px;
+  }
+
+  #name {
+    width: 447px;
+    height: 48px;
+    display: block;
+    margin: 12px 496px 0 496px;
+    color: #172b4d;
+    font-size: 17px;
+    background-color: #fff;
+    line-height: 20px;
+  }
+
+  #email {
+    width: 447px;
+    height: 48px;
+    display: block;
+    margin: 12px 496px 0 496px;
+    color: #172b4d;
+    font-size: 17px;
+    line-height: 20px;
+    background-color: #fff;
+  }
+
+  #msg {
+    width: 447px;
+    height: 170px;
+    border: 1px solid #cfd8dc;
+    background-color: #fff;
+    font-size: 15px;
+    line-height: 24px;
+    color: #b3bac5;
+    margin: 12px 496px 0 496px;
+  }
```

Comment:
- [ ] Let's add the placeholder for the `textarea` in the `HTML` so it matches Figma's design.

Yours:
<img width="266" alt="Loom_iVFAbwDloW" src="https://user-images.githubusercontent.com/81468675/149596189-51829e3f-1e8a-41d8-b917-c8e73eb7b20c.png">

Figma
<img width="182" alt="Loom_NXn8DGYHEx" src="https://user-images.githubusercontent.com/81468675/149596219-4d462476-f9aa-4dce-bc2b-636d1c700ab2.png">
:

Reply:
This was as a result of the closing tag of the textarea not being on the same line as the opening tag. It has been fixed. Thanks